testaddon.go

```diff
@@ -8,9 +8,18 @@ import (
 	"github.com/bitrise-io/go-utils/log"
 )
 
-// getUniqueDir returns the unique subdirectory inside the test addon export diroctory for a given artifact.
-func getUniqueDir(path string) (string, error) {
-	log.Debugf("getUniqueDir(%s)", path)
+func getExportDir(artifactPath string) string {
+	dir, err := getVariantDir(artifactPath)
```

Comment:
Does the test results contents XML need to be validated to be a valid jUnit format?

Reply:
Good question and it actually raises a few other ones.

At this point, I'm not sure if anything not related to Android unit tests should be exported from this step in the first place. 🤔

Reply:
@lpusok and @lszucs the step's responsibility is only to move its outputs to the right directory for the test addon and add the necessary metadata.

After that deploy to bitrise step takes care of the report conversion and upload.